### PR TITLE
Change date for 2019, disable 2018 elements

### PR DESCRIPTION
**Pull request by patricksanders**

### CODE REVIEW

**Comment by brimstone** [comment]
Cut the bit about the theme

**Comment by brimstone** [comment]
Logo needs the year removed or updated, your pick

**Comment by patricksanders** [comment]
I updated the logo, but it must be cached in a CDN or something because I'm still being served the 2018 one. Should fall off eventually though.

File: themes/agency/static/img/logo.svg [hunk 1]
```diff
@@ -21,7 +21,7 @@
             <text x="1409.79px" y="209.155px" style="font-family:'Futura-Medium', 'Futura', sans-serif;font-weight:500;font-size:210.767px;fill:white;">A</text>
         </g>
         <g transform="matrix(0.410944,0,0,0.410944,992.799,166.714)">
-            <text x="978.64px" y="198.071px" style="font-family:'Futura-Medium', 'Futura', sans-serif;font-weight:500;font-size:210.767px;fill:white;">2<tspan x="1097.13px 1215.62px 1334.11px " y="198.071px 198.071px 198.071px ">018</tspan></text>
+            <text x="978.64px" y="198.071px" style="font-family:'Futura-Medium', 'Futura', sans-serif;font-weight:500;font-size:210.767px;fill:white;">2<tspan x="1097.13px 1215.62px 1334.11px " y="198.071px 198.071px 198.071px ">019</tspan></text>
```
Comment (brimstone) [inline themes/agency/static/img/logo.svg:24]: huh

**Comment by brimstone** [comment]
No twitter card?

File: themes/agency/static/img/logo.svg (hunk 1 above)
Comment (patricksanders) [reply themes/agency/static/img/logo.svg:24]: Right?

**Comment by patricksanders** [comment]
Twitter card is there, just way smaller than before for some reason

**Comment by brimstone** [comment]
Something still looks funny with the Files Changed, but I suppose we can fix it in post.

**Pull request merged**